MyHashMap.java

```diff
@@ -50,11 +50,7 @@ public int size(){
 
     // could be wrong
     public boolean contains(K obj){
-        int index = obj.hashCode();
-        if (hashArray[index] == null){
-            return false;
-        }
-        return true;
```

Comment:
this is pretty straightforward, utilize the `contains` function in the MyHashSet, if the `MyHashSet keySet` contains the object, then it is in the `MyHashMap`